actionview/lib/action_view/railtie.rb

```diff
@@ -18,13 +18,25 @@ class Railtie < Rails::Engine # :nodoc:
     config.eager_load_namespaces << ActionView
 
     config.after_initialize do |app|
-      ActionView::Helpers::FormTagHelper.embed_authenticity_token_in_remote_forms =
-        app.config.action_view.delete(:embed_authenticity_token_in_remote_forms)
+      if app.config.action_view.key?(:embed_authenticity_token_in_remote_forms)
```

Comment:
This needs to be deprecated on the module, not in the initializer:
https://github.com/rails/rails/blob/e82b645262b2c4e499428d95cbb99e0352c977eb/actionview/lib/action_view/helpers/form_tag_helper.rb#L26-L27

Reply:
@zzak thanks for the feedback. I've pushed up a change to incorporate this feedback.

Was https://github.com/rails/rails/pull/54369 opened to *replace* this PR, or to supplement it?

Reply:
The plan would be to transfer ownership